proc_dash/app.py

```diff
@@ -1,356 +1,25 @@
-        dataset_name_dialog,
-        html.Div(
-            id="output-data-upload",
-            children=[
-                html.H4(id="input-filename"),
-                dbc.Row(
-                    [
-                        dbc.Col(
-                            html.Div(
-                                children=[
-                                    html.Div(
-                                        id="upload-message",  # NOTE: Temporary placeholder, to be removed once error alert elements are implemented
-                                    ),
-                                    html.Div(
-                                        id="column-count",
-                                    ),
-                                    html.Div(
-                                        id="matching-participants",
-                                        style={"margin-left": "15px"},
-                                    ),
-                                    html.Div(
-                                        id="matching-records",
-                                        style={"margin-left": "15px"},
-                                    ),
-                                ],
-                                style={"display": "inline-flex"},
-                            ),
-                            align="end",
-                        ),
-                        dbc.Col(
-                            dataset_summary_card,
-                        ),
-                    ]
-                ),
-                overview_table,
-            ],
-            style={"margin-top": "10px", "margin-bottom": "10px"},
-        ),
-        dbc.Row(
-            [
-                dbc.Row(filter_form_title),
-                dbc.Row(
-                    [
-                        dbc.Col(
-                            session_filter_form,
-                            width=3,
-                        ),
-                        dbc.Col(
-                            dbc.Row(
-                                id="pipeline-dropdown-container",
-                                children=[],
-                            )
-                        ),
-                    ]
-                ),
-            ],
-            id="advanced-filter-form",
-            style={"display": "none"},
-        ),
-        status_legend_card,
-        dbc.Row(
-            [
-                # NOTE: Legend displayed for both graphs so that user can toggle visibility of status data
-                dbc.Col(
-                    dcc.Graph(
-                        id="fig-pipeline-status", style={"display": "none"}
-                    )
-                ),
-                dbc.Col(
-                    dcc.Graph(
-                        id="fig-pipeline-status-all-ses",
-                        style={"display": "none"},
-                    )
-                ),
-            ],
-        ),
-    ],
-    style={"padding": "10px 10px 10px 10px"},
-)
+app.layout = serve_layout()
```

Comment:
:cherries: 
```suggestion
app.layout = construct_layout()
```
or similar.